db/seeds.rb

```diff
@@ -12,103 +12,120 @@
+                             categories: [seafood_cat, entree_cat],
+                             image: File.open('public/Scyllamari.jpg'))
 
 griffin_wings  = Item.create(title: "Buffalo Griffin Wings",
                              description: "These little wings pack an especially powerful--maybe even majestic--punch",
                              price: 6.00,
                              status: "active",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/BuffaloGriffinWings.jpg'))
 
 pegasus_wings  = Item.create(title: "Buffalo Pegasus Wings",
                              description: "One little bit will set your tastebuds aflight",
                              price: 6.00,
                              status: "retired",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/BuffaloPegasusWings.jpg'))
 
 leg_of_griffin = Item.create(title: "Leg of Griffin",
                              description: "Suffice it to say the king of beasts makes a meal fit for kings",
                              price: 9.50,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/LegOfGriffin.jpg'))
 
 leg_of_pegasus = Item.create(title: "Leg of Pegasus",
                              description: "This dish will send you galloping off to tasty bliss",
                              price: 9.50,
                              status: "retired",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/LegOfPegasus.jpg'))
 
 hydra          = Item.create(title: "Hydra Vodka",
                              description: "Careful with this one! We serve it so virulent your friends may appear to have many heads",
                              price: 6.00,
                              status: "active",
-                             categories: [drinks_cat])
+                             categories: [drinks_cat],
+                             image: File.open('public/HydraVodka.jpg'))
 
 rainbow_pie          = Item.create(title: "Ogre Pie",
                              description: "We hope your appetite is as voracious as this dish's namesake",
                              price: 5.50,
                              status: "active",
-                             categories: [dessert_cat])
+                             categories: [dessert_cat],
+                             image: File.open('public/OgrePie.jpg'))
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.